elblog.go

```diff
@@ -26,9 +26,18 @@ type Log struct {
 	UserAgent                        string
 	SSLCipher                        string
 	SSLProtocol                      string
+	TargetGroupARN                   string
+	TraceID                          string
+	DomainName                       string
+	ChosenCertARN                    string
+	MatchedRulePriority              string
+	RequestCreationTime              time.Time
+	ActionsExecuted                  string
```

Comment:
Based on the description it feels like this should be `[]string`

> The actions taken when processing the request, enclosed in double quotes. This value is a comma-separated list that can include the following possible values: waf, waf-failed, authenticate, redirect, fixed-response, and forward. If no action was taken, such as for a malformed request, this value is set to -.

I can't say I have strong opinions about it, though

Reply:
I kept this as a string because that's how Athena currently treats it. We could change it; I also don't have a strong opinion about this.